next-netflix/app/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,43 @@
+"use client"
+import styled from 'styled-components';
+import NavItem from './NavItem';
+
+
+const Navigation = () =>{
+    return(
+        <Container>
+            <NavBar>
+                <NavItem icon = "Home"/>
+                <NavItem icon = "Search"/>
+                <NavItem icon = "Coming Soon"/>
+                <NavItem icon = "Downloads"/>
+                <NavItem icon = "More"/>
+            </NavBar>
+        </Container>
+    );
+}
+
+export default Navigation;
+
+const Container = styled.div`
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  width: 375px;
+  height: 48px;
+  z-index: 100;
+  background-color: #121212;
+  
+`
+
+const NavBar = styled.div`
+  display: flex;
+  width: 90%;
+  margin-left: 5%;
+  margin-top: 11px;
```

Comment:
margin, padding과 같이 상하좌우 4개의 영역을 지정할 수 있는 css는 위 -> 오른쪽 -> 아래 -> 왼쪽의 순서로 한줄에 적을 수 있습니다.
```suggestion
  margin: 11px 0 0 5%;
```